src/material/grid-list/public-api.ts

```diff
@@ -10,3 +10,5 @@ export * from './grid-list-module';
 export * from './grid-list';
 export * from './grid-tile';
 
+// Privately exported for the grid-list harness.
+export {TileCoordinator as ɵTileCoordinator} from './tile-coordinator';
```

Comment:
I don't think we have a convention for private exports yet. Using this character (similar to framework) has the benfit that it does not easily show up in IDE's, and additionally it won't show up in `ts-api-guardian`. ts-api-guardian does not support aliased exports, so it's important that this import is filtered out.

Reply:
I think this is okay, it seems like it would be pretty rare.

Reply:
I think we do have some private exported symbols floating around (e.g. https://github.com/angular/components/blob/master/src/material/menu/public-api.ts#L13) that start with `_`, maybe just go with that to avoid introducing a new convention?

Reply:
ts-api-guardian will not filter these out. We could configure it to exclude such from goldens, but that will involve more work and I actually think the `e` symbol is fine.

I think we could decide on one convention. I don't mind too much. Let me know what you all prefer?

Reply:
I'm fine with `ɵ` being the convention, I just don't want to have 2 competing conventions

Reply:
The funky character should probably win

Reply:
Yeah, but in order to move with this PR forward we need to decide on a new convention. This seems out ot scope IMO. I do not want to update all goldens in this PR, nor switch all other private exports to `e` in this PR.

I can mark this as blocked and do the changes separately before.

Reply:
I think its fine to merge this one and go back through to update the `_` ones later. I just wanted to make sure that there's a plan to make them all consistent one way or the other

Reply:
That sounds good to me. I will send a PR to make all private exports use the `e` character

Reply:
Just add it to the Jira backlog for now

Reply:
Created COMP-268